Updating extension with new workflows (#7941)
fishtown-analytics/docs.getdbt.com: website/snippets/_install-dbt-extension.md

6. You're all set up! See [about the dbt extension](/docs/about-dbt-extension) for more information on how to use the dbt extension.
    <Lightbox src="/img/docs/extension/kitchen-sink.png" width="60%" title="Showing lineage and compiled code in the extension"/>

## Getting started

> Once the dbt VS Code extension has been installed in your environment, the dbt logo will appear on the sidebar. This provides you with a menu that contains workflows to help you get started, offers information about the extension and your dbt project, and provides helpful links to guide you. For more information, see the [the dbt extension menu](/docs/about-dbt-extension#the-dbt-extension-menu) documentation. 

Reviewer: We use left-side menu [here](https://docs-getdbt-com-git-extension-dbt-labs.vercel.app/docs/install-dbt-extension#upgrade-to-fusion). Suggest we keep it consistent.
```suggestion
Once the dbt VS Code extension has been installed in your environment, the dbt logo will appear in the left-side menu. From here, you can access workflows to help you get started, offers information about the extension and your dbt project, and provides helpful links to guide you. For more information, see the [the dbt extension menu](/docs/about-dbt-extension#the-dbt-extension-menu) documentation. 
```

Author: The sidebar can be custom placed and might not always be on the left, we should change it in the other spot

> Once the dbt VS Code extension has been installed in your environment, the dbt logo will appear on the sidebar. This provides you with a menu that contains workflows to help you get started, offers information about the extension and your dbt project, and provides helpful links to guide you. For more information, see the [the dbt extension menu](/docs/about-dbt-extension#the-dbt-extension-menu) documentation. 

Author: ```suggestion
Once the dbt VS Code extension has been installed in your environment, the dbt logo will appear on the sidebar. From here, you can access workflows to help you get started, offers information about the extension and your dbt project, and provides helpful links to guide you. For more information, see the [the dbt extension menu](/docs/about-dbt-extension#the-dbt-extension-menu) documentation. 
```

To get started with the extension:

> 1. Click on the dbt logo to open the menu and expand the **Get started** section. 

Reviewer: adding this in case they jump right to the steps.
```suggestion
1. From the VS Code left-side menu, click the dbt logo to open the menu and expand the **Get started** section. 
```

> 1. Click on the dbt logo to open the menu and expand the **Get started** section. 

Author: ```suggestion
1. From the sidebar menu, click on the dbt logo to open the menu and expand the **Get started** section. 
```

> 1. From the sidebar menu, click on the dbt logo to open the menu and expand the **Get started** section. 

Reviewer: ```suggestion
1. From the sidebar menu, click the dbt logo to open the menu and expand the **Get started** section. 
```

> 2. Click on the **dbt Walkthrough** status bar to view the welcome screen.

Reviewer: ```suggestion
2. Click the **dbt Walkthrough** status bar to view the welcome screen.
```
